sdk/src/store.rs

```diff
@@ -3877,6 +3912,22 @@ impl Store {
             .map(|(manifest_bytes, _)| Store::from_jumbf(&manifest_bytes, validation_log))?
     }
 
+    /// Handles remote manifests asynchronously when fetch_remote_manifests feature is enabled
+    /// Required because wasm-bindgen cannot use async functions in a sync context.
```

Comment:
Looks like a lot of gymnastics like there to properly get the async/sync functions for each target is going on.

Reply:
Yes. Async is tricky and wasm-bindgen only makes it more so.